--- dvbtune-0.5/dvbtune.c.omv~	2016-03-20 17:32:35.969536946 +0100
+++ dvbtune-0.5/dvbtune.c	2016-03-20 17:33:46.299324992 +0100
@@ -39,6 +39,7 @@
 // Linux includes:
 #include <stdio.h>
 #include <stdlib.h>
+#include <stdint.h>
 #include <string.h>
 #include <ctype.h>
 #include <sys/ioctl.h>
@@ -1405,7 +1406,8 @@ int main(int argc, char **argv)
   }
 
   if (do_monitor) {
-        int32_t strength, ber, snr, uncorr;
+        int32_t strength, snr;
+	uint32_t ber, uncorr;
         fe_status_t festatus;
 
         if((fd_frontend = open(frontenddev[card],O_RDONLY)) < 0){
